ext/pycall/pycall.c

```diff
@@ -1013,6 +1016,9 @@ pycall_call_python_callable(PyObject *pycallable, int argc, VALUE *argv)
   }
 
   res = pyobject_call(pycallable, args, kwargs); /* New reference */
+  pycall_Py_DecRef(pycallable);
+  pycall_Py_DecRef(args);
+  pycall_Py_DecRef(kwargs);
```

Comment:
We need to check whether `kwargs` has a valid `PyObject` pointer.
```c
if (kwargs) {
  pycall_Py_DecRef(kwargs)
}
```

Reply:
Oh YES, we must check whether kwargs is NULL or not.